Insight_Project_Framework/doc_type_classifier.py

```diff
@@ -0,0 +1,62 @@
+import pickle
+import os
+import re
+from sklearn.feature_extraction.text import TfidfVectorizer
+from sklearn import svm
+from sklearn import preprocessing
+
+def read_pickle(text_folder_path, file_name):
+
+    with open(os.path.join(text_folder_path, file_name), 'rb') as filehandle:
+        # read the data as binary data stream
+        read_text = pickle.load(filehandle)
+    return read_text
+
+
```

Comment:
Is this all we need to clean up the text? We can add other functions later on if needed. For now, it's fine.

Reply:
For now I haven't added too much text preprocessing, except for removing new lines and several other line separation expressions. I will modify and fine-tune theses preprocessing steps this week.